src/bin/ion/commands/beta/schema/validate.rs

```diff
@@ -1,6 +1,7 @@
 use crate::commands::IonCliCommand;
 use anyhow::{Context, Result};
 use clap::{Arg, ArgAction, ArgMatches, Command};
+use ion_rs::element::writer::TextKind;
```

Comment:
🗺️ Following changes for schema subcommand are unrelated to code generation. These changes are required due to v0.10.0 of `ion-schema` which includes ISL model changes necessary for code generation.